packages/cli/generators/openapi/spec-helper.js

```diff
@@ -196,7 +196,7 @@ function buildMethodSpec(controllerSpec, op, options) {
   const paramNames = {};
   if (parameters) {
     args = parameters.map(p => {
-      const name = escapeIdentifier(p.name);
+      let name = escapeIdentifier(p.name);
```

Comment:
This is suspicious, why is it necessary to change from `const` to `let`?

Reply:
I'll check.